versioned_docs/version-1.0/language/attachments.mdx

```diff
@@ -120,33 +110,44 @@ access(all) attachment SomeAttachment for SomeContract.SomeStruct {
 }
 ```
 
-By default, the `base` reference is unauthorized, and thus entitled-access members on the base type are inaccessible in the attachment.
-If the author of the attachment wishes to have access to entitled-access members on the base type,
-they must declare that their attachment requires certain entitlements to the base, using `require entitlement E` syntax.
-Required entitlements must be valid entitlements for the base type,
-and requiring entitlements in the attachment declaration will impose additional requirements when the attachment is attached,
-as described below. So, for example:
+Within an attachment's member function, the `base` and `self` references are entitled to the same entitlements that the function's access modifier specifies.
+E.g., in an attachment declared as `access(all) attachment A for R`, within a definition of a function `access(E) fun foo()`, 
```

Comment:
Can an attachment declaration be defined with access modifiers other than `access(all)`? What implications does that have on the permissions / type of `base` and `self`? Maybe add an example

Reply:
No, not anymore.

Reply:
Maybe make it explicit, e.g. something like "Just like composite declarations, attachment declarations must use the access(all) access modifier"